### ISSUE TITLE
Onboard contra costa

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/health#300]
- [x] Get CBO account (UGH WORKING ON IT)
- [ ] Setup case tracking
- [x] Adwords UP

**Comment by lippytak** [comment]
Status: CBO account still not approved (not a blocker). Verbal OK on case tracking via BCW confirmation #s but haven't tested it yet. Joel is reviewing getcalfresh.org and we'll submit 1 case ASAP to get momentum going and test case tracking.

**Comment by daguar** [comment]
CBO account seems to work! @lippytak double check, but it seems to be good to go on the test I ran just now so I'm marking it.

**Comment by lippytak** [comment]
Hi Rebecca, Hi Vakil, Hi Kim,

I'm out for the next couple weeks (on vacation!) so just wanted to give you a quick update on the pilot and make sure you had Alan's contact info (CC'd).

1) After a bit of a delay I'm happy to report that WE'RE LIVE IN CONTRA COSTA! Woohoo! The Food Bank submitted 11 CalFresh applications this week, including helping out some pretty large families and a couple individuals. They were able to help people apply using their tablet onsite at a WIC clinic. I spoke to them afterwards and they were "really impressed, super excited" and most importantly found it "really easy to include the attachments for verifications." Good news! On average it took them about 15 minutes per application, though they did one application in 7 minutes flat.

2) Important changes. Since we first spoke we've made some important changes to hopefully make life easier for clients and counties. First, we've rebranded and redesigned our website to GetCalFresh.org so it makes more sense for clients. Second, we've hooked up GetCalFresh.org to MBCW, so new applications will come directly to you through MBCW like any other online application. All of the cases will be associated with our MBCW CBO account 'codeforamerica'. Hopefully this saves you all some time!

3) Let's start tracking. Here's a spreadsheet that we can use to track a little bit of data about these cases: [REDACTED LINK]

It would be fantastic if we could get some data on pending cases every couple weeks we can keep an eye on issues and see if we need to change the website to help clients submit better applications (or let them know they probably aren't eligible). Since there is no PII in here I think we're totally kosher on privacy but of course let me know if you have any concerns. Vakil, let's chat about this sometime before EOD Monday.

Okay I think that's it for now...have a great couple weeks and talk to you all soon!

Best,
Jake
